packages/node-build-scripts/package.json

```diff
@@ -56,7 +56,7 @@
         "ts-jest": "^29.0.5"
     },
     "engines": {
-        "node": ">=19.3"
+        "node": ">=18.13"
```

Comment:
I picked a relatively recent version of v18 (Jan 2023) here while avoiding constraining this range to the most recent release.